activerecord/test/cases/calculations_test.rb

```diff
@@ -102,6 +102,12 @@ def test_should_group_by_summed_field
     assert_equal 60,   c[2]
   end
 
+  def test_should_qualify_fields_by_group
+    assert_nothing_raised do
+      AuditLog.joins(:developer).group(:id).count
```

Comment:
Can we check the value to make sure that this is doing the right thing? Not raising an exception doesn't mean that there isn't necessarily some issue